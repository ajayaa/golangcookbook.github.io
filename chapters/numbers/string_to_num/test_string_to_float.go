package main

import (
	"fmt"
	"os"
	"strconv"
)

func main() {
	str := "123.123"
	/*
		The first argument to ParseInt is the string to be converted.
		The second argument is the number of bits to which the number should fit.
	*/
	i, err := strconv.ParseFloat(str, 64)
	if err != nil {
		fmt.Println(err)
		os.Exit(1)
	}
	fmt.Printf("%T %f\n", i, i)
}
