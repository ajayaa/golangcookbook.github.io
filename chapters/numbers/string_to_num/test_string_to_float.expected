float64 123.123000
